daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Engine.scala

```diff
@@ -246,7 +248,7 @@ class Engine(val config: EngineConfig = new EngineConfig(LanguageVersion.StableV
         case speedy.Compiler.PackageNotFound(_) =>
           handleMissingDependencies.flatMap(_ => start)
```

Comment:
When do we get missing deps here? If compilation errors are internal errors, presumably we already typechecked/validated before. Shouldn’t that have pulled in the deps making this an internal error?

Reply:
Yes.